app/src/main/java/ch/epfl/sdp/peakar/user/services/providers/firebase/FirebaseSocialList.java

```diff
@@ -0,0 +1,123 @@
+package ch.epfl.sdp.peakar.user.services.providers.firebase;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+
+import java.util.Comparator;
+import java.util.List;
+
+import ch.epfl.sdp.peakar.database.Database;
+import ch.epfl.sdp.peakar.social.SocialItem;
+import ch.epfl.sdp.peakar.social.SocialListAdapter;
+
+/**
+ * This class is an helper class designed to provide <code>SocialActivity</code>
+ * with a synchronized list.
+ */
+public class FirebaseSocialList {
+
+    private static final String USERNAME = "username";
+    private static final String SCORE = "score";
+    private static final String PROFILE_PICTURE = "";
```

Comment:
I know we cannot show the image at the moment, but, however, maybe we should already retrieve the URLs. Or do we leave it for later?